cuda_bindings/pixi.toml

```diff
@@ -0,0 +1,125 @@
+# SPDX-FileCopyrightText: Copyright (c) 2024-2025 NVIDIA CORPORATION & AFFILIATES. All rights reserved.
+#
+# SPDX-License-Identifier: Apache-2.0
+
+[workspace]
+channels = ["conda-forge"]
+platforms = ["linux-64", "linux-aarch64", "win-64"]
+preview = ["pixi-build"]
+
+[workspace.build-variants]
+python = ["3.10.*", "3.11.*", "3.12.*", "3.13.*", "3.14.*"]
+
+[target.linux.tasks.build-cython-tests]
+cmd = ["$PIXI_PROJECT_ROOT/tests/cython/build_tests.sh"]
+
+[target.linux.tasks.test]
+cmd = ["pytest", "$PIXI_PROJECT_ROOT"]
+depends-on = [{ task = "build-cython-tests" }]
+
+[dependencies]
+cuda-bindings = { path = "." }
+libnvjitlink = "*"
+cuda-nvrtc-dev = "*"
+cuda-profiler-api = "*"
+cuda-nvvm = "*"
+
+[target.linux.dependencies]
+libcufile-dev = "*"
+
+[feature.test.dependencies]
+pytest = ">=6.2.4"
+pytest-benchmark = ">=3.4.1"
+pyglet = ">=2.1.9"
+numpy = "*"
+
+[feature.cython-tests.dependencies]
+cython = "*"     # for tests that exercise APIs from cython
+setuptools = "*" # for distutils
+gxx = "*"        # to compile the generated code
+
+# these are necessary because running the cython tests requires compiling
+# *after* the package is built
+[feature.cython-tests.target.linux-64.dependencies]
+cuda-crt-dev_linux-64 = "*"
+
+[feature.cython-tests.target.linux-aarch64.dependencies]
+cuda-crt-dev_linux-aarch64 = "*"
+
+[feature.cython-tests.target.win-64.dependencies]
+cuda-crt-dev_win-64 = "*"
+
+[feature.cu13.system-requirements]
+cuda = "13"
+
+[feature.cu13.dependencies]
+cuda-version = "13.*"
+
+[feature.cu12.system-requirements]
+cuda = "12"
+
+[feature.cu12.dependencies]
+cuda-version = "12.*"
```

Comment:
I am a bit confused by `cu12` here and below. `cuda-bindings` in the main branch does not work with CUDA 12, so how does this work ?